### ISSUE TITLE
Drifts 14: differences in median pressure on the same location

### ISSUE THREAD

**Issue by DavorJ** [DOV-Vlaanderen/groundwater-logger-validation#59]
This is a preliminary analysis to compare barometers on the same location to see what effect the medians have on altitude calculation. Given enough data, I would expect the medians to be around the same pressure (and thus altitude). Something like this:

![image](https://user-images.githubusercontent.com/4161918/78061852-21552580-738e-11ea-8a4e-ee5cf5584e54.png)

BAOL006X_179843 and BAOL006X_555569 also seem OK based on #51, although the former seems a bit jumpy at the end.

The altitude difference based on the medians is computed using barometric formula from #53 and should only be indicative. 

These two match perfectly and are also perfect according to #51: 

![image](https://user-images.githubusercontent.com/4161918/78062839-8f4e1c80-738f-11ea-8196-4874f93b38c5.png)

While in this case the difference is too high:

![image](https://user-images.githubusercontent.com/4161918/78061974-4a75b600-738e-11ea-849b-6766303f745b.png)

But both barometers are broken anyway according to #51. 

The more interesting cases are the ones which are both OK according to #51, but which are off in this analysis. For example:

![image](https://user-images.githubusercontent.com/4161918/78063087-ebb13c00-738f-11ea-84c2-819f7e30a1ad.png)

![image](https://user-images.githubusercontent.com/4161918/78063293-4054b700-7390-11ea-940f-58d6ec20bf47.png)

![image](https://user-images.githubusercontent.com/4161918/78063513-a2152100-7390-11ea-9137-cf38d53d12b8.png)

This could indicate wrong units. On the other hand I think I have too little observations to make any real conclusions. I think the analysis would be better if done based on offset comparison of the red line from analysis #51 or #57.

All the plots are available [here](https://github.com/DOV-Vlaanderen/groundwater-logger-validation/files/4411018/location_comparison.pdf).

**Comment by DavorJ** [comment]
# Drifts 15: Behaviour of median in function of x observations

To verify my concerns above about the median, I assume the KNMI series has perfect measurements and is always on the same altitude. Subsequently, I bootstrap its medians in function of the # of observations, and observe how they behave:

![image](https://user-images.githubusercontent.com/4161918/78073401-2374af80-73a1-11ea-94db-8262d155d6a4.png)

About 1500 observations (=2 years) are needed to reduce the standard error of the median pressure to +/- 1 cmH2O. Note that from analysis #53, based on the barometric formula, we know that 1 cmH2O corresponds to almost 10 m altitude change. So based on 2 years of pressure data our uncertainty of the median is about +/- 2 cmH2O (95% conf.), which is equivalent to an error of approximately 40 m in altitude.

In other words, analysis #59 is mostly useless once we take these expected errors of the median in function of the number of observations into account.
